libs/libarchfpga/src/physical_types.h

```diff
@@ -1382,6 +1382,11 @@ enum e_directionality {
     UNI_DIRECTIONAL,
     BI_DIRECTIONAL
 };
+enum e_parallel_axis {
```

Comment:
Comment what this enum is for and what each value in it means.

Reply:
Added comments accordingly.